### PR TITLE
NO-ISSUE: Clean dnsmasq files

### PR DESCRIPTION
**Pull request by lalon4**
QE CI machines having hundreds of dnsmasq files. This PR clean this file as part of nodes teardown

### CODE REVIEW

**Comment by lalon4** [comment]
Deleting all directory will also delete files that were not created by test-infra. At least on QE machines, we have few flows to run tests (not only test-infra) and therefore we have more files that we wouldn't want to delete

File: src/tests/base_test.py [hunk 1]
```diff
@@ -474,6 +474,10 @@ def get_nodes_func(tf_config: BaseTerraformConfig, cluster_config: ClusterConfig
                 log.info(f"--- TEARDOWN --- deleting iso file from: {_cluster_config.iso_download_path}\n")
                 utils.run_command(f"rm -f {_cluster_config.iso_download_path}", shell=True)
                 self.teardown_nat(_nat)
+                with suppress(FileNotFoundError):
```
Comment (eliorerz) [inline src/tests/base_test.py:477]: What about https://github.com/openshift/assisted-test-infra/blob/e8b6c88414e4c50db1459878eea9cb64f587eaeb/src/tests/base_test.py#L358?

**Comment by michaellevy101** [comment]
> t-infra) and therefore we have more files that we wouldn't want to delete
ok. so maybe delete all dnsmasq files when deleting all resources

File: src/tests/base_test.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (lalon4) [reply src/tests/base_test.py:481]: Added to prepare_nodes, thik its enough?

**Pull request merged**